src/app/shell/sidenav/sidenav.component.ts

```diff
@@ -51,4 +63,9 @@ export class SidenavComponent implements OnInit {
     );
     this.accountMenuItems = accountMenu(() => this.oidcFacade.signoutRedirect());
   }
+  /* eslint-disable @typescript-eslint/no-unused-vars */
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  private onClickSidenav(event: any): void {
```

Comment:
eslint dobrze godo, jak parametr niepotrzebny to chyba lepiej go wywalić i zostawić `onClickSidenav()`?

Reply:
w sumie to dobrze godo, ja to tutaj dałem bo będzie trzeba niedlugo dorzucić jakiś sposób wyświetlania dialogów w zależności od tego co się kliknęło, ale w sumie to chyba lepiej będzie napisać do tego osobne funkcje i już odpowiednio je podłączać pod command

Reply:
wywale to